Lib/test/test_genericalias.py

```diff
@@ -31,6 +34,9 @@ def test_subscriptable(self):
                   Reversible,
                   Container, Collection,
                   Callable,
+                  Mailbox, _PartialFile,
```

Comment:
`_PartialFile` seems to be the object that is actually used, so that is why I am testing it instead of `_ProxyFile`